README.md

```diff
@@ -4,12 +4,12 @@ NRG UI is the design system maintained by the Knoxville Utilities Board. It is b
 
 ## Usage
 
-While the collection of raw CSS is available for use by non-Ember applications, the [Ember addon](./packages/ember/) provides a more integrated experience for Ember applications. To use the CSS in a non-Ember application, check the [design system](./packages/design-system/) for instructions.
+While the collection of raw CSS is available for use by non-Ember applications, the [Ember addon](./packages/ember-core/README.md) provides a more integrated experience for Ember applications. To use the CSS in a non-Ember application, check the [design system](./packages/design-system/) for instructions.
```

Comment:
You don't have to point straight to the README since it's rendered automatically.
```suggestion
While the collection of raw CSS is available for use by non-Ember applications, the [Ember addon](./packages/ember-core/) provides a more integrated experience for Ember applications. To use the CSS in a non-Ember application, check the [design system](./packages/design-system/) for instructions.
```